packages/renderer/src/Route.ts

```diff
@@ -24,3 +27,53 @@
  *            change what the 'current' page is.
  */
 export type NavigationHint = 'root' | 'details' | 'tab';
+
+export const navigationHandle = (page: NavigationPage, parameters?: Record<string, string>) => {
+  switch (page) {
+    case NavigationPage.CONTAINERS:
+      router.goto('/containers');
+      break;
+    case NavigationPage.CONTAINER:
+      parameters && router.goto(`/containers/${parameters['id']}/`);
```

Comment:
either we say that checking only parameters is not enough as we can come with parameters and another field that id
or if we don't care then you can use optional field ? and remove `parameters &&`